integrations/heap/lib/index.js

```diff
@@ -21,7 +21,9 @@ var toString = Object.prototype.toString; // in case this method has been overri
 var Heap = (module.exports = integration('Heap')
   .global('heap')
   .option('appId', '')
-  .tag('<script src="//cdn.heapanalytics.com/js/heap-{{ appId }}.js">'));
+  .option('hostname', 'cdn.heapanalytics.com')
+  .option('options', {})
```

Comment:
The heap JS docs list some of the options, but there are additional options which are not documented. That leads me to believe that it might be best to just provide a single options object, since the allowed options might be account-specific and it might be confusing to provide some configuration options here that aren't available to all accounts. However, I'm happy to split this out into specific configuration options.